src/Encode.fs

```diff
@@ -472,38 +504,25 @@ If you can't use one of these types, please pass an extra encoder.
                     // Evaluate lazily so we don't need to generate the encoder for null values
                     let encoder = lazy
                                     t.GenericTypeArguments.[0]
-                                    |> autoEncoder extra caseStrategy skipNullField
+                                    |> autoEncoder extra skipNullField
                                     |> option
                                     |> boxEncoder
                     boxEncoder(fun (value: obj) ->
                         if isNull value then nil
                         else encoder.Value value)
                 elif fullname = typedefof<obj list>.FullName
-                    || fullname = typedefof<Set<string>>.FullName then
+                    || fullname = typedefof<Set<string>>.FullName
+                    || fullname = typedefof<HashSet<string>>.FullName then
                     // Disable seq support for now because I don't know how to implements to on Thoth.Json.Net
                     // || fullname = typedefof<obj seq>.FullName then
```

Comment:
@alfonsogarciacaro Can I abuse a bit of your presence and ask you if you know-how to implement seq support in Thoth.Json.Net ? 🙏 😄 

I was able to do it for Thoth.Json but unfortunately not for Thoth.Json.Net...

Reply:
I saw the comment and did try to implement support for `seq` too but as you said it wasn't straightforward to do in .Net (don't remember exactly what was the issue) and I quickly gave up to avoid spending too much time with the PR. But I'll give it another go :)